packages/core/src/config/__tests__/load.test.ts

```diff
@@ -117,42 +125,41 @@ describe('loadConfig', () => {
 
 describe('findConfig', () => {
   it('should find redocly.yaml', async () => {
-    jest.spyOn(fs, 'existsSync').mockImplementation((name) => name === 'redocly.yaml');
+    vi.spyOn(fs, 'existsSync').mockImplementation((name) => name === 'redocly.yaml');
     const configName = findConfig();
     expect(configName).toStrictEqual('redocly.yaml');
   });
   it('should find .redocly.yaml', async () => {
-    jest.spyOn(fs, 'existsSync').mockImplementation((name) => name === '.redocly.yaml');
+    vi.spyOn(fs, 'existsSync').mockImplementation((name) => name === '.redocly.yaml');
     const configName = findConfig();
     expect(configName).toStrictEqual('.redocly.yaml');
   });
   it('should throw an error when found multiple config files', async () => {
-    jest
-      .spyOn(fs, 'existsSync')
-      .mockImplementation((name) => name === 'redocly.yaml' || name === '.redocly.yaml');
+    vi.spyOn(fs, 'existsSync').mockImplementation(
+      (name) => name === 'redocly.yaml' || name === '.redocly.yaml'
+    );
     expect(findConfig).toThrow(`
       Multiple configuration files are not allowed.
       Found the following files: redocly.yaml, .redocly.yaml.
       Please use 'redocly.yaml' instead.
     `);
   });
   it('should find a nested config ', async () => {
-    jest.spyOn(fs, 'existsSync').mockImplementation((name) => name === 'dir/redocly.yaml');
-    jest.spyOn(path, 'resolve').mockImplementationOnce((dir, name) => `${dir}/${name}`);
+    vi.spyOn(fs, 'existsSync').mockImplementation((name) => name === 'dir/redocly.yaml');
+    vi.spyOn(path, 'resolve').mockImplementationOnce((dir, name) => `${dir}/${name}`);
     const configName = findConfig('dir');
     expect(configName).toStrictEqual('dir/redocly.yaml');
   });
 });
 
 describe('getConfig', () => {
-  jest.spyOn(fs, 'hasOwnProperty').mockImplementation(() => false);
   it('should return empty object if there is no configPath and config file is not found', () => {
+    vi.spyOn(Object.prototype, 'hasOwnProperty').mockImplementation(() => false);
```

Comment:
Is it testing [this line](https://github.com/Redocly/redocly-cli/pull/1998/files?diff=split&w=1#diff-91233afeb25fea2c651ad22452d823830a0c6e8378deba81a1109b57cdae3fb6R95)? If yes, then should we remove/change it?